app/assets/stylesheets/common/_markus.scss

```diff
@@ -350,25 +352,25 @@ fieldset {
 }
 
 .disable {
-  color: $dark-grey;
+  color: $line;

```

Comment:
I just thought about this here, but I think we should probably have a constant `$disabled` or something, so that it's not strictly tied to line colour. (Would be worth reviewing all the colours that represent "disabled" elements.)